examples/js/loaders/GLTF2Loader.js

```diff
@@ -2111,6 +2112,92 @@ THREE.GLTF2Loader = ( function () {
 						meshNode = new THREE.Mesh( geometry, material );
 						meshNode.castShadow = true;
 
+						if ( primitive.targets !== undefined ) {
+
+							geometry.morphTargets = [];
+
+							var targets = primitive.targets;
+							var morphAttributes = geometry.morphAttributes;
+
+							for ( var i = 0, il = targets.length; i < il; i ++ ) {
+
+								var target = targets[ i ];
+								geometry.morphTargets.push( { name: 'morphTarget' + i } );
+
+								if ( target.POSITION !== undefined ) {
+
+									material.morphTargets = true;
+
+									if ( morphAttributes.position === undefined ) morphAttributes.position = [];
+
+									// Three.js morph formula is
+									//   position
+									//     + weight0 * ( morphTarget0 - position )
+									//     + weight1 * ( morphTarget1 - position )
+									//     ...
+									// while the glTF one is
+									//   position
+									//     + weight0 * morphTarget0
+									//     + weight1 * morphTarget1
+									//     ...
+									// then adding position to morphTarget.
+									// motphTarget should depend on mesh, so cloning attribute.
```

Comment:
*morphTarget

These BufferAttributes aren't mutated while playing animation, are they? If not, what's the reason for cloning them? I'm not sure I follow the "should depend on mesh" part.